VimeoNetworkingExample-iOS/VimeoNetworkingExample-iOS/DetailViewController.swift

```diff
@@ -35,23 +32,6 @@ class DetailViewController: UIViewController {
         super.viewDidLoad()
         // Do any additional setup after loading the view, typically from a nib.
         self.configureView()
-        
```

Comment:
how about displaying a feature of VIMVideo e.g. `label.text = detail.videoDescription` in `func configureView()`